assets/js/modules/analytics/components/settings/SettingsMain.test.js

```diff
@@ -60,6 +60,11 @@ describe( 'SettingsMain', () => {
 			{ body: fixtures.accountsPropertiesProfiles, status: 200 }
 		);
 
+		fetchMock.get(
+			/tagmanager\/data\/settings/,
+			{ body: {}, status: 200 },
+		);
+
```

Comment:
Why not receive the settings instead?

```js
registry.dispatch( MODULES_TAGMANAGER` ).receiveGetSettings( {} );
```

Just curious if there's a particular reason you chose to take this approach instead.

Reply:
Ah.. yes, thanks! Updated.